service/user/internal/src/main/resources/liquibase/0.2.0/user.sql

```diff
@@ -34,7 +34,7 @@ CREATE TABLE usr_user (
 
   PRIMARY KEY (id),
   CONSTRAINT usr_uc_name UNIQUE (name)
-) DEFAULT CHARSET=utf8;
+) ENGINE = InnoDB DEFAULT CHARSET = utf8;
```

Comment:
Yes this works :) Tested on demo machine and build on Travis. 
H2 and MySQL have a different way/keywords to define `COLLATION` so I was not able to specify it. I would like to use `COLLATION = utf8_unicode_ci`.

Did you have any problem with this?